src/appshell/qml/Preferences/internal/ProgrammeStartSection.qml

```diff
@@ -27,7 +27,7 @@ import MuseScore.UiComponents 1.0
 BaseSection {
     id: root
 
-    title: qsTrc("appshell/preferences", "Program start")
+    title: qsTrc("appshell/preferences", "Programme start")
```

Comment:
"Program" was correct. We use American English in all strings, but even in British English it is normal to use the American spelling when referring to computer programs.

Reply:
Hmmm... I tried to fix 13044 - Lyrics default to lyricist text-style. This is an old (rejected) PR...
Would you mind killing this PR, I'll make a new one?

Reply:
It's possible to drop the other changes and fix only the lyricist bug in this PR, but feel free to close it and make a new one if you prefer.